cogeo_mosaic/scripts/cli.py

```diff
@@ -92,6 +108,7 @@ def create(
         quadkey_zoom=quadkey_zoom,
         minimum_tile_cover=min_tile_cover,
         tile_cover_sort=tile_cover_sort,
+        tilematrixset=tilematrixset,
```

Comment:
the default will be `None` and will only be set when passing `--tms ...` 

internally, cogeo-mosaic will default to WebMercatorQuad but without setting it in the MosaicJSON file